ntbs-service/Pages/Notifications/Edit/Episode.cshtml.cs

```diff
@@ -76,6 +81,8 @@ protected override async Task<bool> ValidateAndSave()
 
             if (!TryValidateModel(Episode, Episode.GetType().Name))
             {
+                // Detach notification to avoid getting cached notification when retrieving from context
+                context.Entry(Notification).State = EntityState.Detached;

```

Comment:
I'm a little uneasy over manual detaching... but if we can't get a workaround for this it'll probably have to do:
Do we want to detach if `TrySetAndValidateDateOnModel` fails too - a quick look shows me that that isn't currently returning true/false but is an easy fix to sort that out.

Reply:
Probably no need to do it on `TrySetAndValidateDateOnModel`, as when it fails, it never sets the date (if i remember correctly) - but worth double checking.

I would suggest explaining *why* getting the cached value is wrong, i.e. that it gets shown not just on the input field, but in the banner, too.